### PR TITLE
Use LookAndMove tool as default walk tool

### PR DESCRIPTION
**Pull request by achrysaetos**
Closes https://github.com/iTwin/viewer/issues/197.

### CODE REVIEW

File: packages/modules/viewer-react/src/components/app-ui/providers/StandardNavigationToolsProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+} from "@itwin/appui-abstract";
+import type { DefaultNavigationTools } from "@itwin/appui-react";
+import { CoreTools, ToolbarHelper, ToolItemDef } from "@itwin/appui-react";
+import { IModelApp } from "@itwin/core-frontend";
+
+/**
+ * Provide standard tools for the ViewNavigationWidgetComposer.
+ * @public
+ */
+export class StandardNavigationToolsProvider extends BaseUiItemsProvider {
```
[the remark this answers is not in the document]
Comment (achrysaetos) [reply packages/modules/viewer-react/src/components/app-ui/providers/StandardNavigationToolsProvider.tsx:24]: Good point.

File: packages/modules/viewer-react/src/components/app-ui/providers/ViewerNavigationToolsProvider.tsx [hunk 2]
```diff
@@ -24,4 +32,42 @@ export class ViewerNavigationToolsProvider extends StandardNavigationToolsProvid
...
+    toolbarOrientation: ToolbarOrientation
+  ): CommonToolbarItem[] {
+    const items: CommonToolbarItem[] = super.provideToolbarButtonItems(
+      stageId,
+      stageUsage,
+      toolbarUsage,
+      toolbarOrientation
+    );
+
+    if (
```
Comment (aruniverse) [inline packages/modules/viewer-react/src/components/app-ui/providers/ViewerNavigationToolsProvider.tsx:49]: do you not get any of the other items now?

File: packages/modules/viewer-react/src/components/app-ui/providers/ViewerNavigationToolsProvider.tsx [hunk 3]
```diff
@@ -24,4 +32,42 @@ export class ViewerNavigationToolsProvider extends StandardNavigationToolsProvid
...
+    ) {
+      items.push(
+        ToolbarHelper.createToolbarItemFromItemDef(
+          10,
+          new ToolItemDef({
+            toolId: "View.LookAndMove",
+            iconSpec: "icon-walk",
+            execute: () =>
+              IModelApp.tools.run(
+                "View.LookAndMove",
```
Comment (aruniverse) [inline packages/modules/viewer-react/src/components/app-ui/providers/ViewerNavigationToolsProvider.tsx:62]: did you see if there was a Command for this?

File: packages/modules/viewer-react/src/components/app-ui/providers/ViewerNavigationToolsProvider.tsx (hunk 2 above)
Comment (achrysaetos) [reply packages/modules/viewer-react/src/components/app-ui/providers/ViewerNavigationToolsProvider.tsx:49]: You mean the toolbar button items? They're still there. I just had to hide the old walk tool and show the new one.
image

File: packages/modules/viewer-react/src/components/app-ui/providers/ViewerNavigationToolsProvider.tsx (hunk 3 above)
Comment (achrysaetos) [reply packages/modules/viewer-react/src/components/app-ui/providers/ViewerNavigationToolsProvider.tsx:62]: Is [this](https://github.com/iTwin/itwinjs-core/blob/112933a965e339f36b8af4f092fae8c73f6767da/core/frontend/src/tools/Tool.ts#L865) the command you mean?

Comment (aruniverse) [reply packages/modules/viewer-react/src/components/app-ui/providers/ViewerNavigationToolsProvider.tsx:62]: Wondering if there is a command for this tool like the old one, see https://github.com/iTwin/itwinjs-core/blob/master/ui/appui-react/src/appui-react/ui-items-provider/StandardNavigationToolsProvider.ts#L64

Comment (aruniverse) [reply packages/modules/viewer-react/src/components/app-ui/providers/ViewerNavigationToolsProvider.tsx:62]: ideally wed use ToolbarHelper.createToolbarItemFromItemDef, and just pass in the command if one exists

Comment (achrysaetos) [reply packages/modules/viewer-react/src/components/app-ui/providers/ViewerNavigationToolsProvider.tsx:62]: You mean one of the commands from [CoreTools](https://github.com/iTwin/itwinjs-core/blob/007be59f93813bc5525b1fe2afc48f20761bca56/common/api/appui-react.api.md?plain=1#L1413), or somewhere else?

Comment (aruniverse) [reply packages/modules/viewer-react/src/components/app-ui/providers/ViewerNavigationToolsProvider.tsx:62]: yea i was wondering if the there was a command (or i guess an ItemDef) for it somewhere in the repo.

looks like its the LookAndMoveTool, but there is no itemDef command for it, so i guess this is fine

**Review by aruniverse — APPROVED** [review]

**Pull request merged**